mikelpardo/mobile-sales
# Add movile list view and add theme

**Description**

Add movile list view with some fake data

**How to test**

1. Go to: http://localhost:3000/
2. Check if movile list view displays correctly with faked data
